Sources/ProjectDescription/Config.swift

```diff
@@ -171,3 +191,39 @@ extension Config.GenerationOptions {
         }
     }
 }
+
+extension Config.GenerationOptions.CodeCoverageMode {
+    enum CodingKeys: String, CodingKey {
+        case all
+        case relevant
+        case targets
+    }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        if container.allKeys.contains(.all), try container.decode(Bool.self, forKey: .all) {
```

Comment:
Can't this be just:
```suggestion
        if container.allKeys.contains(.all) {
```

The value is always true in any case

Reply:
Yes it can, but other `GenerationOptions` follow this pattern so I don't see a reason why this shouldn't, or is it something that would be likely to be removed?